ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-release-1.0.yaml

```diff
@@ -345,6 +363,42 @@ tests:
         requests:
           cpu: 300m
           memory: 500Mi
+- always_run: false
+  as: test-e2e-disconnected
+  steps:
+    cluster_profile: gcp
```

Comment:
You can't use gcp here. I guess the `gcp-observability` is the only profile for this.